Kernel/WaitQueue.cpp

```diff
@@ -71,50 +71,56 @@ void WaitQueue::wake_one()
     m_wake_requested = !did_unblock_one;
 }
 
-void WaitQueue::wake_n(u32 wake_count)
+u32 WaitQueue::wake_n(u32 wake_count)
 {
     if (wake_count == 0)
-        return; // should we assert instead?
+        return 0; // should we assert instaed?
```

Comment:
you repleace "instead" with "instaed" here.

Reply:
whoops. Not sure how that happened...

Reply:
@tomuta: ping. `lint-clang-format.sh` is failing.